src/client/store/mapbox/index.js

```diff
@@ -30,14 +30,17 @@ export const actions = {
     state.mapObject.map.on(event, handler)
     commit('addEventHandler', { event, handler })
   },
+  addOnceEventHandler({state, commit}, { event, handler }) {
+    state.mapObject.map.once(event, handler)
+  },
   initMap({ commit }, map) {
     map.on('load', () => {
       commit('setMapIsLoaded', true)
       commit('initMap', map)
     })
   },
   moveMapToCenter({ state }) {
-    state.mapObject.map.flyTo({center:[ NL_LAT, NL_LNG ], zoom: MAP_ZOOM})
+    state.mapObject.map.flyTo({center:[ NL_LNG, NL_LAT ], zoom: MAP_ZOOM})
```

Comment:
In `map-config.js` you use `NETHERLANDS_CENTER_LATITUDE` while here you use abbrevations. Whichever you pick, try to be consistent.